### PR TITLE
STANDARDS-3  Initial set of terms, definitions, and a dummy icon 

### PR DESCRIPTION
**Pull request by nedhorning**
Created an initial set of terms, definitions, and a dummy icon for review and testing.

### CODE REVIEW

**Comment by beccaRND** [comment]
@nedhorning looking at the commits sidebar list I'm seeing - symbols and some blocks are deleted. Does this mean anything significant or just GitHub being Github?

**Comment by nedhorning** [comment]
@beccaRND I expect this is GitHub being GitHub. It's a little confusing since there are so many changes although most are either a file being deleted (red -) or added (green +). For edited files, deleted text is highlighted in red, and added text is highlighted in green. Does that help?

**Comment by aaronc** [comment]
One issue here is that the deploy preview isn't working so we can't see the live pages. I will try to fix that

File: README.md
Comment (beccaRND) [inline README.md:17]: So in this model you could have the same sub directory items under different directory items, such as coastal for marine and coastal for wetlands. that doesn't present a problem though because it's organized under a different directory item, is that correct?

File: src/content/activity/Conservation.md
Comment (beccaRND) [inline src/content/activity/Conservation.md:6]: could conservation include an area or resource that is not used or actively managed, more simply preserved or protected? if that is the case then it could be different from sustainable use and management.

File: README.md
Comment (nedhorning) [reply README.md:17]: The situation you describe is possible but my take is that it should be discouraged/rejected if someone proposed that. Unless, of course, we want coastal to show up under marine and wetlands. At this point, I'd say that's for the approvers to decide using "due diligence." I expect these sorts of questions will pop up from time to time.

File: src/content/impact/ImprovedBiodiversity.md
Comment (beccaRND) [inline src/content/impact/ImprovedBiodiversity.md:6]: the definition is correct for the md Improved Biodiversity but the title says increased carbon sequestration and storage

File: src/content/impact/ImprovedWaterInfiltration.md
Comment (beccaRND) [inline src/content/impact/ImprovedWaterInfiltration.md:6]: because infiltration is measured as a rate, should the definition include "enhance the rate and ability of water..."

**Review by beccaRND — APPROVED** [review]
Comments left for your review; most are suggested changes. The only comment that must be addressed is the Improved biodiversity field having the wrong title.

File: src/content/activity/Conservation.md
Comment (S4mmyb) [reply src/content/activity/Conservation.md:6]: Wondering the same. @nedhorning maybe it would make more sense to define it as, 

The protection, preservation, management, or restoration of natural environments and the ecological 
 communities that inhabit them. Conservation is generally held to include the sustainable use and management of natural resources including wildlife, water, air, and earth deposits, to ensure their availability to present and future generations.

File: src/content/activity/SoilAmendments/Biochar.md
Comment (S4mmyb) [reply src/content/activity/SoilAmendments/Biochar.md:6]: but the end effect is still the same, right? @beccaRND did you have a proposed upgrade? Or should we keep it as is

File: src/content/activity/SoilAmendments.md
Comment (S4mmyb) [reply src/content/activity/SoilAmendments.md:6]: I think it's probably fine as is personally

File: src/content/impact/AvoidedDeforestationDegradation.md
Comment (S4mmyb) [reply src/content/impact/AvoidedDeforestationDegradation.md:6]: standing forests usually refer to old groth though, correct? I think we want to support all forests

File: src/content/impact/ImprovedBiodiversity.md
Comment (S4mmyb) [inline src/content/impact/ImprovedBiodiversity.md:2]:
```suggestion
title: Improved Biodiversity
```

File: src/content/impact/ImprovedBiodiversity.md
Comment (S4mmyb) [reply src/content/impact/ImprovedBiodiversity.md:6]: suyggested change above

File: src/content/impact/ImprovedWaterInfiltration.md
Comment (S4mmyb) [inline src/content/impact/ImprovedWaterInfiltration.md:6]:
```suggestion
Enhance the rate and ability of water to soak into the ground, improving groundwater recharge and reducing surface runoff. 
```

**Review by S4mmyb — COMMENTED** [review]
Hey @nedhorning , overall i think this looks great! Becca had a few comments for suggested changes. Once those are corrected we can approve and merge. Nice job!

File: src/content/activity/Conservation.md
Comment (nedhorning) [reply src/content/activity/Conservation.md]: I followed Sam's definition but removed "restoration" since that's covered in EcosystemRestoration.

File: src/content/activity/SoilAmendments.md
Comment (nedhorning) [reply src/content/activity/SoilAmendments.md:6]: I added: Soil amendments should not be confused with fertilizer which is meant to supply nutrients to the soil.

File: src/content/activity/SoilAmendments/Biochar.md
Comment (nedhorning) [reply src/content/activity/SoilAmendments/Biochar.md:6]: I left this as is for the reason Sam highlights, but can change it if you think it should explicitly state that biochar does not directly impact soil quality or something like that.

File: src/content/activity/SoilAmendments/EnhancedRockWeathering.md
Comment (nedhorning) [reply src/content/activity/SoilAmendments/EnhancedRockWeathering.md:6]: I could use an example. I'm not sure what you mean by "fast-track". Do you mean relative to the carbon removal rate before it was crushed? Even after it's crushed it's still kinda slow a removing carbon. I'm trying to not go too deep into the weeds with definitions unless it's necessary to add clarity.

File: src/content/impact/AvoidedDeforestationDegradation.md
Comment (nedhorning) [reply src/content/impact/AvoidedDeforestationDegradation.md:6]: I prefer to leave this as is. My interpretation of standing forests is forests with trees growing as opposed to harvested. I think "forest" typically  creates a mental image of an area with trees.

File: src/content/impact/ImprovedBiodiversity.md
Comment (nedhorning) [reply src/content/impact/ImprovedBiodiversity.md:6]: I changed this to the correct title

File: src/content/impact/ImprovedForestHealth.md
Comment (nedhorning) [reply src/content/impact/ImprovedForestHealth.md:6]: I added: human interventions, and other environmental stressors

File: src/content/impact/ImprovedWaterInfiltration.md
Comment (nedhorning) [reply src/content/impact/ImprovedWaterInfiltration.md:6]: I added "rate" as you suggested

File: src/content/impact/ReducedFertilizerAmendments.md
Comment (nedhorning) [reply src/content/impact/ReducedFertilizerAmendments.md:6]: I added " and yields"

File: src/content/activity/Agroforestry/RiparianForestBuffers.md
Comment (nedhorning) [reply src/content/activity/Agroforestry/RiparianForestBuffers.md:6]: Yes - I mistakenly put that under Agroforestry. I moved the file up one level

File: src/content/activity/SoilAmendments/Biochar.md
Comment (beccaRND) [reply src/content/activity/SoilAmendments/Biochar.md:6]: in the end the effect is not the same if nutrients are not present or added. my proposed upgrade would be to say "adding biochar to the soil to improve its quality and nutrient and water holding capacity."

Comment (nedhorning) [reply src/content/activity/SoilAmendments/Biochar.md]: I like that. I changed the text as you suggested

File: src/content/activity/SoilAmendments/EnhancedRockWeathering.md
Comment (S4mmyb) [reply src/content/activity/SoilAmendments/EnhancedRockWeathering.md:6]: I think its fine to leave as is.

**Review by S4mmyb — APPROVED** [review]

**Pull request merged**
